docker-compose.test.yaml

```diff
@@ -19,12 +19,16 @@ services:
       - ./jest.config.js:/home/node/jest.config.js
       - ./package.json:/home/node/package.json
     environment:
-      AZURE_STORAGE_CONNECTION_STRING: DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://ffc-pay-batch-processor-azurite:10000/devstoreaccount1;
+      AZURE_STORAGE_CONNECTION_STRING: DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://ffc-pay-batch-azurite:10000/devstoreaccount1;
       DISABLE_SEQUENCE_VALIDATION: ${DISABLE_SEQUENCE_VALIDATION:-false}
 
   ffc-pay-batch-processor-postgres:
     volumes:
       - postgres_data:/var/lib/postgresql/data
 
+  ffc-pay-batch-azurite:
+    labels:
+      com.docker.compose.pay.role: test
```

Comment:
start script will check label to see if storage already running.  Want to ignore if test though.

Labels follow same pattern as default docker compose labels